src/targets/csharp/restsharp/client.ts

```diff
@@ -20,8 +20,15 @@ export const restsharp: Client = {
       return 'Method not supported';
     }
 
+    function toPascalCase(str: string): string {
+      return str.replace(
+        /\w+/g,
+        word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase(),
+      );
+    }
+
     push(`var client = new RestClient("${fullUrl}");`);
-    push(`var request = new RestRequest(Method.${method.toUpperCase()});`);
+    push(`var request = new RestRequest(Method.${toPascalCase(method)});`);
```

Comment:
I jumped the gun on this. This also doesn't work because `RestRequest` expects a string not an enum. At a minimum I'll need a `ToString()`. I'll need to check the RestSharp docs. I'm sure they have a recommended practice.

Reply:
This should be 

* Initialize the `RestClient` with a base url
* Initialize the `RestRequest` with two parameters: relative url and `Method`.

Reply:
Unfortunately I don't see that the `request` preserves base url and relative url. So I guess we can just initialize the client with full url and pass `null` as the relative url.